tempto-examples/src/main/resources/tempto-configuration.yaml

```diff
@@ -84,3 +84,9 @@ list:
     - ala
     - ma
     - kota
+
+ldap:
```

Comment:
Yes, we are planning to get OpenLDAP on docker next sprint. Once we have that I can add some example tests in a new patch. Do you want me to revert the change in this file?